docs/android/v2/how--to-guides/handle-interruptions/interruption-handling.mdx

```diff
@@ -13,3 +13,15 @@ When a telephonic voice call comes in and the user accepts the following happens
 When the call ends the SDK will restore all videos, taking care not to un-mute you if you had muted yourself before the call came in. Neither will it un-mute all the peers if you had muted them from the app.
 
 No code required, this happens automatically for all calls.
+
+
+| **App State** | **Scenario**                               | **Android**                                |

```

Comment:
I think just clearly mentioning what our sdk does is enough. Background and foreground is not needed, since sdk does not do anything specific related to it
We can just mention:
`SDK mutes local audio and video track if a phone call interruption happens
The SDK restores back the state to original state after the interrupting phone call is over`